src/index.ts

```diff
@@ -116,6 +175,60 @@ export class BKTClientImpl implements Bucketeer {
     }, this.config.pollingIntervalForRegisterEvents!);
   }
 
+  async stringVariation(user: User, featureId: string, defaultValue: string): Promise<string> {
+    return (await this.stringVariationDetails(user, featureId, defaultValue)).variationValue;
+  }
```

Comment:
Can you move this right after the `booleanVariationDetails`?

Reply:
updated https://github.com/bucketeer-io/node-server-sdk/pull/70/commits/1884c252600c7182406c4f7630e96785702f2451